x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+package org.elasticsearch.xpack.core.ilm.action;
+
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+
+public enum ILMActions {
```

Comment:
```
joegallo@simulacron:~/Code/elastic/elasticsearch $ git grep '/ilm/' | grep -vi test/ | grep core/ilm/action
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/DeleteLifecycleAction.java:    public static final String NAME = "cluster:admin/ilm/delete";
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ExplainLifecycleAction.java:    public static final String NAME = "indices:admin/ilm/explain";
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/GetLifecycleAction.java:    public static final String NAME = "cluster:admin/ilm/get";
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/GetStatusAction.java:    public static final String NAME = "cluster:admin/ilm/operation_mode/get";
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java:    public static final ActionType<AcknowledgedResponse> START = ActionType.acknowledgedResponse("cluster:admin/ilm/start");
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java:    public static final ActionType<AcknowledgedResponse> STOP = ActionType.acknowledgedResponse("cluster:admin/ilm/stop");
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java:    public static final ActionType<AcknowledgedResponse> RETRY = ActionType.acknowledgedResponse("indices:admin/ilm/retry");
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java:    public static final ActionType<AcknowledgedResponse> MOVE_TO_STEP = ActionType.acknowledgedResponse("cluster:admin/ilm/_move/post");
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/ILMActions.java:    public static final ActionType<AcknowledgedResponse> PUT = ActionType.acknowledgedResponse("cluster:admin/ilm/put");
x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/action/RemoveIndexLifecyclePolicyAction.java:    public static final String NAME = "indices:admin/ilm/remove_policy";
```

What makes these five special enough to warrant their membership here, while the other five get the usual treatment and aren't treated specially? (Editiorial note: I'm only like 15% of the way through reviewing this PR, so maybe I'll end up running into the answer along the way.)

Reply:
These are used in the core (test) codebase where their transport action isn't available. We shouldn't have that kind of circular(ish) dependency situation to begin with but I couldn't find a short enough way of cleaning all that up so I went for something basic like this here for now :)